src/Mizar/MizarMainWindow.ui

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MainWindow</class>
+ <widget class="QMainWindow" name="MainWindow">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>986</width>
+    <height>449</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>MainWindow</string>
+  </property>
+  <widget class="QWidget" name="centralwidget">
+   <layout class="QVBoxLayout" name="verticalLayout">
```

Comment:
you might want to consider having more "concrete" names for the vertical and horizontal layouts. The tree nature of the UI elements won't be easily reflected in the generated class. You'll end up with `main_window.ui->verticalLayout`, `main_window.ui->verticalLayout_2`, or `main_window.ui->horizontalLayout_2`. It'll be hard to know which one corresponds to what layout.